src/components/Home.js

```diff
@@ -141,5 +252,4 @@ function Home() {
   );
 }
 
-export default Home;
-
+export default Home;
```

Comment:
Looks like this is missing a newline too :)

Reply:
Updated!